### PR TITLE
vendor quickstart templates into origin

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@smarterclayton @csrwng ptal.  I am not loving this mechanism for automatically parsing the readme and pulling down the files, but it could be worse...

open to reasonable suggestions for improvement.

Creating a directory for each one makes the scripting much more complicated.

File: hack/update-external-examples.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+pushd examples/quickstarts
+rm *json
+rm *yaml
+curl `grep https://raw.githubusercontent.com README.md | sed -E "s/.*\((.*)\) -.*/\\1 -O/"`
```
Comment (smarterclayton) [inline hack/update-external-examples.sh]: Will this work on Mac?

**Comment by smarterclayton** [comment]
I'm ok with it not creating directories.

File: hack/update-external-examples.sh (hunk 1 above)
Comment (csrwng) [inline hack/update-external-examples.sh]: It will work

**Comment by smarterclayton** [comment]
LGTM, add a section to HACKING.md to describe it.

**Comment by smarterclayton** [comment]
And comments in the script.

**Comment by bparees** [comment]
@smarterclayton updated.

**Comment by smarterclayton** [comment]
LGTM [merge]

**Pull request merged**